README.md

```diff
@@ -104,3 +104,29 @@ or in short
 ```bash
 just c
 ```
+
+## Regtest environment
+
+The following command will start a regtest bitcoin node, electrs, chopsticks, esplora, lnd node and a http server allowing you to faucet your bitcoin and lightning wallet.
+
+```bash
+docker-compose up
+```
+
+### How to faucet your lightning wallet.
+
+#### Setup
+
+1. Open `http://localhost:8080/faucet`
+2. Ensure you have enough balance on your bitcoin wallet. Hit the mine button a couple of times if not.
+3. Get a new address of your coordinator by running `curl http://localhost:8000/api/newaddress`
+4. Faucet some coins to your coordinator wallet. Hit the mine button afterwards so the transaction gets into a block.
+5. Open `http://localhost:8080/channel`
```

Comment:
This tries to forward me to `http://localhost:80/channel` which fails because that port is not exported:

```
curl http://localhost:8080/channel -v

*   Trying 127.0.0.1:8080...
* Connected to localhost (127.0.0.1) port 8080 (#0)
> GET /channel HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.86.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 301 Moved Permanently
< Server: nginx/1.23.3
< Date: Sun, 26 Feb 2023 23:42:21 GMT
< Content-Type: text/html
< Content-Length: 169
< Location: http://localhost/channel/
< Connection: keep-alive
<
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.23.3</center>
</body>
</html>
* Connection #0 to host localhost left intact
* ```

Because of this I can't test any further.

Reply:
I had the same problem using `curl`. But I could get around it using a web browser _if_ I included the trailing forward slash i.e. `http://localhost:8080/channel/`.

Reply:
thanks @luckysori : this fixes it.

Reply:
I've updated the `README.md`